src/test/java/co/com/api/test/BodyDto/PersonDto.java

```diff
@@ -0,0 +1,16 @@
+package co.com.api.test.BodyDto;
```

Comment:
change BodyDto to dto (remember that the namespaces in java is in lower)

Reply:
Done